### PR TITLE
Blog app - controllers

### PR DESCRIPTION
**Pull request by mperezbazan**
In this PR I follow these requirements:
- [x] Take a look at the wireframes below. Make sure that your app handles the URLs used in the wireframes: 
<p align="center">
  All users page wireframe
  Single user page wireframe
  All posts by a given user page wireframe
  Single post page wireframe
</p>

-  For each URL you should create:
    - [x] A route.
    - [x] An action in the correct controller.
    - [x] A view file.
- [x] View files should include only placeholders written in plain HTML.
- [x] Make sure that when you run the `rails routes` command in your console, you can see the following output
image

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+  resources :users do
+    resources :posts
+  end
```
Comment (prolajumokeoni) [inline config/routes.rb]: - [ ] Please do not expose unnecessary endpoints in your `routes.rb`, make sure only routes needed for users and posts are created, see example below  ⬇️  

```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @mperezbazan 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Controllers and views setup correctly 👍🏿 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (mperezbazan) [reply config/routes.rb]: Thanks for your review!

**Review by levy002 — APPROVED** [review]
## Hello @mperezbazan 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**